src/main/java/org/isf/menu/gui/Login.java

```diff
@@ -192,14 +198,21 @@ public LoginPanel(Login myFrame) {
 				OHServiceExceptionUtil.showMessages(e1);
 				System.exit(1);
 			}
-
-			usersList = new JComboBox<>();
-			for (User u : users) {
-				usersList.addItem(u.getUserName());
+			
+			if(usersListLogin) {
+				usersList = new JComboBox<>();
+				for (User u : users) {
+					usersList.addItem(u.getUserName());
+				}
+	
+				Dimension preferredSize = usersList.getPreferredSize();
+				usersList.setPreferredSize(new Dimension(120, preferredSize.height));
+			}
+			else {
```

Comment:
Project formatting standards puts the `else {` on the same line as the previous `}'